### PR TITLE
Test new panel

### PR DESCRIPTION
**Pull request by MatteoStifano**
New e2e test that perform the following steps:
- Create new host via Rest API;
- Create new panel, filter by host name and select time-usage template;
- Remove created host.

### CODE REVIEW

File: tests/cypress/e2e/spec.cy.ts [hunk 1]
```diff
@@ -32,4 +33,33 @@ describe('Source configuration', () => {
...
+    cy.get('input[id="input_Host"]').click();
+    cy.contains(hostName).click({ force: true }); // Hostname -> <current host>
+
+    cy.get('input[id="input_Template"]').type('{downArrow}{enter}'); // Template -> Time usage by phase
+
+    cy.get('button[title="Apply changes and save dashboard"]').contains('Save').click({ force: true });
+    cy.get('input[aria-label="Save dashboard title field"]').type(' ' + randID);
+    cy.get('button[aria-label="Save dashboard button"]').click();
+    cy.reload();
+
```
Comment (BenediktSeidl) [inline tests/cypress/e2e/spec.cy.ts:61]: do we need to verify that something actually happened? or do we just check for javascript errors?

**Comment by MatteoStifano** [comment]
@BenediktSeidl I introduced an assertion over the panel legend to make sure a graph is actually plotted.
Also, I had to find a solution regarding test flakyness in the CI (see iteration for the input template). As second step, it would be useful to generalize such solution so that it can be used for other tests too. I think for the time being we can live with this.

File: tests/cypress/e2e/spec.cy.ts [hunk 2]
```diff
@@ -33,4 +34,52 @@ describe('Source configuration', () => {
...
+    cy.get('input[id="react-select-7-input"]').type('Host name{enter}'); // Filter -> 'Host name'
+
+    cy.get('input[id="input_Host"]').type('{enter}'); // Hostname -> <current host> (one entry only)
+    cy.contains(hostName).should('be.hidden');
+
+    cy.get('[class="panel-content"]').should('be.visible');
+
+    function select_time_usage_template(count: number = 20) {
+      // selecting a template results in a flaky behavior
+      // TODO: fix flakyness and remove this iteration
```
Comment (BenediktSeidl) [inline tests/cypress/e2e/spec.cy.ts:59]: currently preparing a patch that could help you there: we changed the dropdown type back to `AsyncSelect` that displays a small spinner while loading. the absence of this spinner would indicate that the dropdowns are  ready.

**Review by BenediktSeidl — APPROVED** [review]
i guess we already talked about this, but just to be sure: in the future we need to verify that the correct number of labels appear in the graphing area.

File: tests/cypress/e2e/spec.cy.ts (hunk 2 above)
Comment (MatteoStifano) [reply tests/cypress/e2e/spec.cy.ts:59]: Ack.

**Pull request merged**